logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java

```diff
@@ -56,37 +59,36 @@ public void setUp() {
 
   @After
   public void tearDown() {
+    // Clean up created logs
+    deleteLog(QUICKSTART_LOG);
```

Comment:
do you want to do this `@before` as well? incase something broke last time?

Reply:
I'm concerned that part of the reasons the tests are failing periodically is that the deletes are actually being registered after the test line is logged. By moving the cleanup to the tearDown, it should always clean up unless the build is somehow forced to stop between the file log and that this exact moment.

Reply:
I'm just concerned that some crash happens and it doesn't execute - so I ask Q's.  If you are happy, I'm happy. (ie you don't think that will happen)

Reply:
sgtm